src/modules/app/containers/App.js

```diff
@@ -10,18 +9,68 @@ import languagesEndpoint from '../../endpoint/endpoints/languages'
 import categoriesEndpoint from '../../endpoint/endpoints/categories'
 import locationEndpoint from '../../endpoint/endpoints/locations'
 import eventsEndpoint from '../../endpoint/endpoints/events'
+import CategoriesPage from 'routes/categories/containers/CategoriesPage'
+import SearchPage from '../../../routes/search/containers/SearchPage'
+import DisclaimerPage from '../../../routes/disclaimer/containers/DisclaimerPage'
+import EventsPage from '../../../routes/events/containers/EventsPage'
+import PdfFetcherPage from '../../../routes/pdf-fetcher/containers/PdfFetcherPage'
+import LandingPage from '../../../routes/landing/containers/LandingPage'
+import MainDisclaimerPage from '../../../routes/main-disclaimer/components/MainDisclaimerPage'
+import RouteConfig from '../RouteConfig'
+import RouterFragment from './RouterFragment'
 
 class App extends React.Component {
+  store
+  routeConfig
+
   componentWillMount () {
-    this._store = createReduxStore(createHistory)
+    const routes = [
+      {
+        path: '/'
+      },
+      {
+        id: SearchPage,
+        path: '/:location/:language/search'
+      },
+      {
+        id: DisclaimerPage,
+        path: '/:location/:language/disclaimer'
+      },
+      {
+        id: EventsPage,
+        path: '/:location/:language/events(/:id)'
+      },
+      {
+        id: PdfFetcherPage,
+        path: '/:location/:language/fetch-pdf'
+      },
+      {
+        id: CategoriesPage,
+        path: '/:location/:language(/*)'
+      },
+      {
+        id: MainDisclaimerPage,
+        path: '/disclaimer'
+      },
+      {
+        path: '/:unknown(/)'
+      },
+      {
+        id: LandingPage,
+        path: '/:language(/)'
```

Comment:
can we unify this with `{path: '/'}` as `{id: LandingPage, path: '/(:language(/))}`, because '/' is also the path to the landing page

Reply:
Those are two different paths. One shows the I18nRedirect and one is LandingPage.